### ISSUE TITLE
Geodesic Distance Measurements

### ISSUE THREAD

**Issue by sanjana-roy** [GIS4DEV/GIS4DEV.github.io#18]
I'm a little confused about the requirement for forcing the algorithm to use geodesic distance measurements. The only place where I see this is applicable is in Output Type in the Distance Matrix. Would this have to do with using a Linear distance matrix versus another type?

**Comment by mtango99** [comment]
"If the input layer uses a projected coordinate system, that system is used or planar distance calculation.
If the input layer uses a geographic coordinate system, the project ellipsoid is used for ellipsoidal distance calculation."
([link to Joe's report](https://gis4dev.github.io/lessons/02b_showmethecode.html))

So... I guess we'd have to specify a CRS for both our input and target layers at the beginning of our models? Is there a geodesic CRS that would work across all datasets?

**Comment by vinfalardeau** [comment]
Yeah this is tricky – I tried reprojecting the input and target layers to WGS 84, but since that's a geographic coordinate system, the distance matrix seems to just ignore it and use whatever is the project ellipsoid. For instance, I ran the model twice, once with WGS 84 as the project ellipsoid and once with GRS 80, and I got slightly different distances. Granted, the differences were about a millionth of meter, but I guess maybe we're stuck with trusting the user to either have the right projected coordinate systems for both layers or the right project ellipsoid?
